### DISCUSSION TITLE
LocalityawareFeedforwardNetwork

### DISCUSSION THREAD

**Discussion by jo-member** [bcaitech1/p4-ocr-hansarang#47] [Show and tell]
본래 baseline code의 nn.Linear를 사용한 feedforward network는 spatial 한 정보를 활용하지 않는 연산 network 였다.
하지만 이미지라는 공간적인 정보가 중요한 우리 data의 특성상 Locality feed forward network를 사용하는게 실험 지표와 LB 모두 더 좋은 결과를 얻어내는것을 확인하였다.

**Comment by remaindere** [comment]
실험 지표 뿐만 아니라, 실제 inference 결과도 향상되지 않았나요? 글에 명시해 주시는 것이 더 좋을 것 같습니다!

**Reply by jo-member** [reply]
수정했습니다 :)
